### ISSUE TITLE
Elasticsearch collector won't run with official elasticsearch package installed

### ISSUE THREAD

**Issue by rtoma** [BrightcoveOS/Diamond#591]
After installing the official Elasticsearch client (https://github.com/elasticsearch/elasticsearch-py) the Diamond ES collector does not work anymore.

Using strace I found the rootcause.

The client package is named 'elasticsearch'. At startup Diamond looks for available collectors in /usr/share/diamond/collectors and starts to load the modules. When attempting to load the elasticsearch collector, it does not read the python files from /usr/share/diamond/collectors/elasticsearch, but instead uses the installed elasticsearch package. It loads the /.../site-packages/elasticsearch/**init**.py, which does nothing.

To summarize: Diamond sees there is a ES collector available, attempt to loads it, reads the wrong file, does not see its not a valid collector and continues on his merry way.

My dirty-fix: rename the collector directory from 'elasticsearch' to 'elasticsearch-diamond'. Same for the collector .py file.

**Comment by kawikao** [comment]
This ended up being a pretty horrible situation. I wrote an ES handler that used the ES client. The horrible part is the handler code also loads the collectors which required me to rename the ES collector.

**Comment by kormoc** [comment]
This repo has moved to https://github.com/python-diamond/Diamond

Please open a new ticket there if this issue is not resolved with the current code there.

Thanks!
